src/library/crate/cratestorage.h

```diff
@@ -280,8 +280,8 @@ class CrateStorage: public SqlStorage {
             CrateId crateId); // no db access
 
     // The name must be escaped with FieldEscaper
-    static QString formatSubselectQueryForCrateTrackIdsByEscapedName(
-                                                              const QString& crateName); 
+    static QString formatQueryForTrackIdsByEscapedName(
```

Comment:
This is a better name, but it's not clear what name should be passed to the function. How about `formatQueryForTrackIdsByCrateName`?